sips/core/voluntary_exit.md

```diff
@@ -0,0 +1,74 @@
+| Author      | Title                          | Category | Status |
+|-------------|--------------------------------|----------|--------|
+| Alon Muroch | Voluntary Exit | Core     | Draft  |
+
+
+**Summary**
+Describes how the ssv operators performs a voluntary exit for a validator.  
+<em>Important to note that a user holding the validator’s private key can sign a voluntary exit as well.</em>
+
+
+**Specification**
+
+**Networking**
+A dedicated subnet for voluntary exits will be added to the network for all exit related communication between operators.
+All operators subscribe to the network.
+
+**Messages**
+New SSV MsgType is introduced
+
+| Name       | Type    | Value | Description                          |
+|------------|---------|-------|--------------------------------------|
+| VoluntaryExitMsgType | MsgType | 4     | SSVMessage type for voluntary exit messages |
+
+The SSVMessage Data field will encode the voluntary exit SignedMessage data structure. 
+
+```go
+type VoluntaryExitMessage struct {
```

Comment:
Why not using `SSVMessage` struct?

Reply:
It does, it's encoded later on in an ssv msg